03_how_ln_works.asciidoc

```diff
@@ -96,7 +96,7 @@ Payment channels have a few very interesting and useful properties:
 
 
 Bitcoin was about five years old when talented developers first figured out how payment channels could be constructed and by now there are at least three different methods known.
-This chapter will focus on the channel construction method first described in the Lightning Network whitepaper by Joseph Poon and Thaddeus Dryja in 2015. These are known as _Poon-Dryja_ channels, and are the channel construction method currently used in the Lightning Network.
+This chapter will focus on the channel construction method first described in the https://lightning.network/lightning-network-paper.pdf[Lightning Network whitepaper] by Joseph Poon and Thaddeus Dryja in 2015. These are known as _Poon-Dryja_ channels, and are the channel construction method currently used in the Lightning Network.
 The other two proposed methods are _Duplex Micropayment_ channels, introduced by Christian Decker around the same time as the "Poon-Dryja" channels and _eltoo_ channels, introduced in 2018 by Christian Decker, Rusty Russel and (author of this book) Olaoluwa Osuntokun.
```

Comment:
maybe we should also link the duplex micro payment channels (https://link.springer.com/chapter/10.1007/978-3-319-21741-3_1) and the eltoo paper (https://blockstream.com/eltoo.pdf) ?

Reply:
@renepickhardt https://tik-db.ee.ethz.ch/file/716b955c130e6c703fac336ea17b1670/duplex-micropayment-channels.pdf this is the free version. springer is paywalled.

Reply:
Please make an issue or a separate pull request for the Duplex Micro Payment paper